packages/next/build/index.ts

```diff
@@ -502,47 +496,30 @@ export default async function build(dir: string, conf = null): Promise<void> {
         buildManifest,
         config.experimental.modern
       )
-      const bundleRelative = path.join(
-        isLikeServerless ? 'pages' : `static/${buildId}/pages`,
-        actualPage + '.js'
-      )
-      const serverBundle = path.join(
-        distDir,
-        isLikeServerless ? SERVERLESS_DIRECTORY : SERVER_DIRECTORY,
-        bundleRelative
-      )
+
+      const serverBundle = getPagePath(page, distDir, isLikeServerless)
 
       let isSsg = false
       let isStatic = false
       let isHybridAmp = false
       let ssgPageRoutes: string[] | null = null
       let hasSsgFallback: boolean = false
 
-      pagesManifest[page] = bundleRelative.replace(/\\/g, '/')
```

Comment:
@Timer @ijjk  I'm not sure why this line was here. It seems unneeded as it overrides the existing pagesManifest path which was generated during the build

Reply:
Yeah it should be un-needed since the correct path should already be in the `pages-manifest` 👍